Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -161,3 +209,89 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
         }
     }
 }
+
+// MARK: Dictionary Colon Rule
+extension ColonRule {
+
+    public func validateFile(_ file: File, kind: SwiftExpressionKind,
```

Comment:
this function name could make it clearer that it only detects dictionary violations